packages/tasks/src/hardware.ts

```diff
@@ -159,7 +159,11 @@ export const SKUS = {
 			"RTX 3060": {
 				tflops: 12.74,
 				memory: [12, 8],
-			},
+			}, 
```

Comment:
```suggestion
			},
```

Just removing a trailing space :)